pkg/services/alerting/notifiers/webhook.go

```diff
@@ -80,6 +80,12 @@ func (wn *WebhookNotifier) Notify(evalContext *alerting.EvalContext) error {
 	bodyJSON.Set("ruleName", evalContext.Rule.Name)
 	bodyJSON.Set("state", evalContext.Rule.State)
 	bodyJSON.Set("evalMatches", evalContext.EvalMatches)
+	bodyJSON.Set("orgId", evalContext.Rule.OrgID)
+	bodyJSON.Set("dashboardId", evalContext.Rule.DashboardID)
+	bodyJSON.Set("panelId", evalContext.Rule.PanelID)
+
+	dashboardUid, _ := evalContext.GetDashboardUID()
```

Comment:
I realize now that `dashboardUid` is fetched from the database by calling `evalContext.GetDashboardUID()`. Didn't consider that earlier and that's not something want to call in a notifier on each notification. We need to fix so that `dashboardUid` is fetched from database when fetching the alert rule, but should probably be fixed in a separate PR.

So not sure the changes in this PR are useful without the `dashboardId`. Can you describe your use case around this?

Reply:
I want to have one server to receive all webhooks. This PR enables the server to tell different dashboards and orgs apart.

Reply:
Make sense. I would suggest reverting the `dashboardUid` changes for us to be able to accept this change.